### PR TITLE
✨ POO - src/api/ - ES6

### PR DESCRIPTION
**Pull request by VGamezz19**
There is new feature for you old code gitHubApi. I had implement POO, chek it and tell me 😁

✔︎ all tests runing fine 
❌no 100% coverage from FetchApi.js

```sh
src
├── api
│   ├── FetchApi.js
│   ├── GitHubApi.js
│   └── index.js
├── index.js
└── utils.js
```

### CODE REVIEW

File: src/api/FetchApi.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    constructor(baseUrl) {
+        this.baseUrl = baseUrl ? baseUrl : "https://api.github.com/";
+    }
+
+    getBaseUrl() {
+        return this.baseUrl;
+    }
+
+    /**
+     * Make a request to GitHub public API
```
Comment (didaquis) [inline src/api/FetchApi.js]: Maybe `Make a request to GitHub public API` needs to change to `Make a request to public API`.

File: src/api/FetchApi.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const fetch = require("node-fetch");
+
+module.exports = class FetchApi {
+    constructor(baseUrl) {
+        this.baseUrl = baseUrl ? baseUrl : "https://api.github.com/";
```
Comment (didaquis) [inline src/api/FetchApi.js]: I'm not sure if it's good idea define this URL on this file. Maybe it's better to do it in the `GitHubApi.js` file

**Review by didaquis — COMMENTED** [review]
Nice PR!

File: src/api/FetchApi.js (hunk 1 above)
Comment (VGamezz19) [reply src/api/FetchApi.js]: 👍

File: src/api/FetchApi.js (hunk 2 above)
Comment (VGamezz19) [reply src/api/FetchApi.js]: yes! you are right!! I'll change it

**Comment by VGamezz19** [comment]
@didaquis  ready!

**Pull request merged**